infra/nncc/CMakeLists.txt

```diff
@@ -40,6 +40,14 @@ macro(nnas_include PREFIX)
 endmacro(nnas_include)
 
 macro(nnas_find_package PREFIX)
+  # The policy CMP0057 supports new ``if()`` IN_LIST operator.
+  # The ``OLD`` behavior for this policy is to ignore the IN_LIST operator.
+  # The ``NEW`` behavior is to interpret the IN_LIST operator.
+  # The libboost 1.74 uses IN_LIST operator in a CMake file for the ``NEW`` behavior.
+  # But CMake version 3.22.1 warns when the policy is not set and uses ``OLD`` behavior.
+  # Let's set it to ``NEW`` explicitly.
+  # This policy requires ``cmake_minimum_required(VERSION 3.3)``.
+  cmake_policy(SET CMP0057 NEW)
```

Comment:
Do we need this policy setting? IMO, it's duplication with above minimum requirement update.
As I understood, https://github.com/Samsung/ONE/issues/9432#issuecomment-1184412692 fails because minimum version is 3.1 but `BoostConfig.cmake` in system uses `cmake-3.3`'s new feature - `if(... IN_LIST ...)`.

Reply:
You are right. I will remove it. Thanks.